### DISCUSSION TITLE
Follow up on previousData in placeholderData ignores cache

### DISCUSSION THREAD

**Discussion by kimwes** [TanStack/query#6365] [Q&A] [answered]
Regarding the previous discussion: https://github.com/TanStack/query/discussions/6340 and the linked issue there. Having followed the whole process I have to say I'm very impressed with the whole flow and interaction of the maintainers. Much appreaciated!
I cloned the github.com/TanStack/query repository and confirmed that the test case now passes. However, I'm still having issues with the same scenario in my own project. I'm using the latest version of react-query (5.8.2) that should have the fix. I've created a minimal example in codesandbox.io to demonstrate the issue: https://codesandbox.io/s/epic-neco-3ngwrt that uses the same code as the test case in the query repository with some minor styling additions. There you just have to manually press the button that the test case would automatically press. It also fails at the same spot. It is as if the repositories pass old code because I looked at the code in the node_modules build folders and it looked like it din't have the changes in the queryObserver file. I tried myself with npm and pnpm and with clearing caches and on different computers with no luck. Is this something that perhaps fixes itself with time when there are new versions and builds?

**Comment by TkDodo** [comment, marked as answer]
I think I screwed up the release somehow. Let me force release another patch.

**Reply by TkDodo** [reply]
done. please try 5.8.3

https://github.com/TanStack/query/releases/tag/v5.8.3

**Reply by kimwes** [reply]
Can confirm that it now works correctly, much appreaciated!
